java/code/src/log4j.properties

```diff
@@ -1,4 +1,4 @@
-appenders = RootAppender, XmlRpcFileAppender, XmlRpcFailOverAppender, FrontendLogControllerAppender, RemoteCommandsAppender
+appenders = RootAppender, XmlRpcFileAppender, XmlRpcFailOverAppender, FrontendLogControllerAppender, RemoteCommandsAppender, HibernateAppender
```

Comment:
Is this appender being configured? The code for it is commented. 
I think we should add it in a new line with a comment, so devs could remember it needs to be added here.